css/.csscomb.json

```diff
@@ -13,11 +13,9 @@
     "combinator-space": [" ", " "],
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": null,
     "rule-indent": "    ",
     "stick-brace": " ",
     "unitless-zero": true,
-    "quotes": false,
```

Comment:
Should this be removed, or just set to our preferred setting? Which [should be `single`](https://github.com/mobify/mobify-code-style/tree/develop/css/css-best-practices#format)